pkg/scorecard.go

```diff
@@ -83,7 +86,7 @@ func getRepoCommitHash(r clients.RepoClient) (string, error) {
 	if len(commits) > 0 {
 		return commits[0].SHA, nil
 	}
-	return "", nil
+	return "", errEmptyRepository
```

Comment:
optional: can we flip this existing code around for "happy path" readability?

```go
if len(commits) == 0 {
	return "", errEmptyRepository
}
return commits[0].SHA, nil
```

Reply:
done